src/org/opendatakit/briefcase/ui/export/components/ConfigurationDialogForm.java

```diff
@@ -0,0 +1,189 @@
+package org.opendatakit.briefcase.ui.export.components;
+
+import static java.awt.event.KeyEvent.VK_ESCAPE;
+import static javax.swing.JComponent.WHEN_ANCESTOR_OF_FOCUSED_COMPONENT;
+import static javax.swing.KeyStroke.getKeyStroke;
+
+import java.awt.Component;
+import java.awt.FlowLayout;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JDialog;
+import javax.swing.JPanel;
+import org.opendatakit.briefcase.ui.reused.WindowAdapterBuilder;
+
+@SuppressWarnings("checkstyle:MethodName")
+public class ConfigurationDialogForm extends JDialog {
+  private JPanel dialog;
+  private JButton okButton;
+  private JButton cancelButton;
+  private JButton removeButton;
+  private JPanel rightActions;
+  private JPanel leftActions;
+  private JPanel actions;
+  private ConfigurationPanelForm configurationPanelForm;
+
+  ConfigurationDialogForm(ConfigurationPanelForm form) {
+    configurationPanelForm = form;
+    $$$setupUI$$$();
+    setContentPane(dialog);
+    setModal(true);
+    getRootPane().setDefaultButton(okButton);
+
+    okButton.addActionListener(e -> dispose());
+    removeButton.addActionListener(e -> dispose());
+    cancelButton.addActionListener(e -> dispose());
+
+    setDefaultCloseOperation(DO_NOTHING_ON_CLOSE);
+
+    addWindowListener(new WindowAdapterBuilder().onClosing(e -> dispose()).build());
+
+    dialog.registerKeyboardAction(e -> dispose(), getKeyStroke(VK_ESCAPE, 0), WHEN_ANCESTOR_OF_FOCUSED_COMPONENT);
+  }
+
+  public void open() {
+    setVisible(true);
```

Comment:
For me, this shows the dialog far away from the main window.

![not centered](https://user-images.githubusercontent.com/55099/34911230-01d92476-f87b-11e7-9d71-6a5e7cb96dbf.png)

If you’ll add this here,
`setLocationRelativeTo(null);`

You can have such beauty as this:
![centered](https://user-images.githubusercontent.com/55099/34911234-17384004-f87b-11e7-9488-82376d4fdf5c.png)

And, I think good taste (if not some style guide somewhere) suggests having a title in all dialogs.

Reply:
Thanks!